File: uploaddemo/src/business/shotRecord/java/com/jw/shotRecord/listener/JCameraListener.java
package com.jw.shotRecord.listener;

import android.graphics.Bitmap;

/**
 * =====================================
 * 作    者: 陈嘉桐
 * 版    本：1.1.4
 * 创建日期：2017/4/26
 * 描    述：
 * =====================================
 */
public interface JCameraListener {

    void captureSuccess(Bitmap bitmap);

    void recordSuccess(String url, Bitmap firstFrame);

    void captureEdiit(Bitmap bitmap);

    void recordEdiit(String url, Bitmap firstFrame);
}

File: uploaddemo/src/business/shotRecord/java/com/jw/shotRecord/listener/JCameraListener.kt
package com.jw.shotRecord.listener

import android.graphics.Bitmap

/**
 * =====================================
 * 作    者: 陈嘉桐
 * 版    本：1.1.4
 * 创建日期：2017/4/26
 * 描    述：
 * =====================================
 */
interface JCameraListener {

    fun captureSuccess(bitmap: Bitmap)

    fun recordSuccess(url: String, firstFrame: Bitmap)

    fun captureEdit(bitmap: Bitmap)

    fun recordEdit(url: String, firstFrame: Bitmap)
}
